### PR TITLE
Feature/gh activity

### PR DESCRIPTION
**Pull request by tlaz4**

### CODE REVIEW

File: posts/templates/author/author_posts.html [hunk 1]
```diff
@@ -15,19 +15,34 @@
...
+                    id="follow-user">Unfollow User
+            </button>
+        {% else %}
+            <button type="button" class="btn btn-info follow-button"
+                    onclick="follow_user('{{ author.id }}', '{{ request.user.id }}')" id="follow-user">Friend User
+            </button>
+        {% endif %}
+    </div>
+    <div class="row top-buffer">
+        {% if user.username == author.username %}
```
Comment (MichaelParadis) [inline posts/templates/author/author_posts.html]: For multi node stuff we should be checking user.id == author.id

File: posts/static/author/author_posts.js [hunk 2]
```diff
@@ -33,3 +33,103 @@ function unfollow_user(id) {
...
+        if(response[i]['id'] == oldId){
+            break;
+        }
+
+        var body = create_github_post(response[i]);
+        
+        fetch('/posts/', {
+            method: 'post',
+            headers: {
+                "X-CSRFToken": csrf,
```
Comment (MichaelParadis) [inline posts/static/author/author_posts.js]: Please add a todo to update to use basic auth. I have changed some things for mr worldwide and will need to update your stuff to get it working

**Review by MichaelParadis — DISMISSED** [review]

**Review by MichaelParadis — CHANGES_REQUESTED** [review]
Please update based on the comments I have made on the other review.

**Comment by tlaz4** [comment]
Refactored JS for posting, github pulling is complete

**Review by MichaelParadis — APPROVED** [review]

**Pull request merged**